### DISCUSSION TITLE
.env

### DISCUSSION THREAD

**Discussion by gevrey** [n3d1117/chatgpt-telegram-bot#51] [Q&A] [answered]
Hi there and great job.

Why isn't the "temperature" part of the .env file too?

**Comment by n3d1117** [comment, marked as answer]
Hi @gevrey! No particular reason, initially I just didn't want to pollute the `.env` file with too many options. 
But i'll probably move it there, together with `n_choices` and `image_size`

EDIT: just added `N_CHOICES`, `TEMPERATURE` and `IMAGE_SIZE` to the `.env` file!
